docs_dev/releasing.md

```diff
@@ -260,6 +262,18 @@ If you aren't already working on a release branch (of the form `v${MAJOR}.${MINO
 2.  they allow sophisticated users to track the development of a release (by using the release branch as a `ksonnet` registry), and
 4.  they simplify backporting critical bugfixes to a patchlevel release particular release stream (e.g., producing a `v0.1.1` from `v0.1-branch`), when appropriate.
 
+### Enable Periodic tests on the release branch
+
+Once the release branch is cut we need to enable periodic tests on the release branch and setup a
+testgrid dashboard
```

Comment:
Add a link to the dashboard? Ex: https://prow.k8s.io/?type=periodic&job=kubeflow-periodic-release-branch

Reply:
Done